.circleci/compare_bytecode_reports.sh

```diff
@@ -30,6 +30,7 @@ native_platforms=(
     ubuntu2004-static
     ubuntu
     osx
+    osx_x86_64
```

Comment:
How about `intel` as a suffix for this? Also `arm` for ARM-only, though it looks like for now we don't need that one.

```suggestion
    osx_intel
```

Reply:
Thought you were nitpicking when first saw the comment, but going through the PR, I actually agree now :) Especially with regards to `native` vs `native-x86-64`.

Reply:
Turned out that there is also `x86_64h`, so I would say its the best to leave it like that. But I renamed it in `scripts/bytecodecompare/prepare_report.py` to just be `x86-64` and `x86-64h`. In circleci config and in `compare_bytecode_reports.sh` script I just left the osx prefix.